src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -205,7 +234,8 @@ public AbstractAlignmentMerger(final File unmappedBamFile, final File targetBamF
                                    final SAMFileHeader.SortOrder sortOrder,
                                    final PrimaryAlignmentSelectionStrategy primaryAlignmentSelectionStrategy,
                                    final boolean addMateCigar,
-                                   final boolean unmapContaminantReads) {
+                                   final boolean unmapContaminantReads,
+                                   final UNMAPPING_READ_STRATEGY unmappingReadsStrategy) {
```

Comment:
Probably need a comment that unmappingReadsStrategy is ignored if unmapContaminantReads is true